velox/vector/ComplexVector.cpp

```diff
@@ -525,6 +525,15 @@ void ArrayVectorBase::copyRangesImpl(
   }
 }
 
+void RowVector::validate(const VectorValidateOptions& options) const {
+  BaseVector::validate(options);
+  for (auto& child : children_) {
+    if (child != nullptr) {
+      child->validate(options);
```

Comment:
Maybe we should do some validation across children too ? same size , and if not trailing nulls etc..

Reply:
Sure, we can do that as a follow up, would you like to make that contribution?